packages/components/src/dropdown-menu-v2-ariakit/styles.ts

```diff
@@ -21,12 +22,15 @@ const ANIMATION_PARAMS = {
 };
 
 const CONTENT_WRAPPER_PADDING = space( 2 );
-const ITEM_PREFIX_WIDTH = space( 7 );
-const ITEM_PADDING_INLINE_START = space( 2 );
-const ITEM_PADDING_INLINE_END = space( 2.5 );
-
-// TODO: should bring this into the config, and make themeable
-const DEFAULT_BORDER_COLOR = COLORS.ui.borderDisabled;
+const ITEM_PADDING_BLOCK = space( 2 );
+const ITEM_PADDING_INLINE = space( 3 );
+
+// TODO:
+// - those values are different from saved variables?
+// - should bring this into the config, and make themeable
+// - border color and divider color are different?
```

Comment:
Flagging this for conversation

Reply:
> border color and divider color are different?

Yup that's fine.

> should bring this into the config, and make themeable

I'm not sure what is entailed, but the components should eventually be themable using the new color scales:

<img width="832" alt="theme" src="https://github.com/WordPress/gutenberg/assets/846565/57092560-95f1-4833-9d18-30f402cadf79">

Reply:
Thank you for clarifying. 

There is one more question from the list above that I wanted to clarify — from the design spec, it seems that the border color is gray 300, and the divider is gray 200. 

While these values seem to align with the inline comments [where the grays are defined](https://github.com/WordPress/gutenberg/blob/dccbb4646c07d1ff748f3114b63ac2a4adcda84c/packages/components/src/utils/colors-values.js#L17-L20), I noticed that we have [some variables specific to borders that are not using the same colors](https://github.com/WordPress/gutenberg/blob/dccbb4646c07d1ff748f3114b63ac2a4adcda84c/packages/components/src/utils/colors-values.js#L42-L45) — I guess it would be good if you could review where that are being used, and potentially give feedback around whether we can tweak those variables to use gray 300 and 200.

Reply:
The second group of vars seem to be applied to the experimental `ToggleGroupControl`. They make sense in the context of an interactive element like a form input where there are stricter a11y requirements for contrast. Visual elements (like the menu container) can use the lighter borders.

Reply:
Good to know, and likely something that we should codify when working on the wider theme / design system